python/triqs_dft_tools/converters/plovasp/proj_shell.py

```diff
@@ -186,6 +176,9 @@ def extract_tmatrices(self, sh_pars):
                 matrix = raw_matrix
 
             ndim = nrow
+            print("ndim = {}".format(ndim))
```

Comment:
Are those prints only for debug, or should we keep them now? If we keep them we can maybe add a sentence what this is?